### PR TITLE
add a joke

### PR DESCRIPTION
**Pull request by Raj-Thombare**
Indian programmers can relate to this joke.

### CODE REVIEW

**Comment by eklavyadev** [comment]
That's a good joke but probably Europeans would not get it

**Comment by Raj-Thombare** [comment]
yes brother but i'm sure like us many indians will also come across this and if they'll see it they would understand!

File: jokes/index.json [hunk 1]
```diff
@@ -2266,5 +2266,11 @@
     "type": "general",
     "setup": "My older brother always tore the last pages of my comic books, and never told me why.",
     "punchline": "I had to draw my own conclusions."
+  },
+  {
+    "id": 389,
+    "type": "programming",
+    "setup": "A male developer often gets called as a Dev, then what would you call a female developer?.",
```
Comment (CodingSpecies) [inline jokes/index.json:2273]: I think the `.` should be removed @Raj-Thombare

**Review by CodingSpecies — CHANGES_REQUESTED** [review]

File: jokes/index.json (hunk 1 above)
Comment (eklavyadev) [reply jokes/index.json:2273]: Yeah that's the main cause

**Comment by eklavyadev** [comment]
> Can you guys help me out here? Is there a chance this could be considered offensive?

No way man! This joke is legit, in Hindi Devi means a female deity

**Comment by 15Dkatz** [comment]
Committed: b183cd81f496310a1f7912c357e52145a4238077

**Pull request closed without merging**